CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) a
 
 ## [0.11.7]
 ### Added
+- Support `shouldFitContainer` in `Tooltip`
```

Comment:
should be in 0.11.8 even if we don't bump the version in this commit because 0.11.7 is already released